packages/bsky/src/data-plane/server/db/migrations/20240723T220703655Z-quotes.ts

```diff
@@ -0,0 +1,23 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('quote')
```

Comment:
We probably want to add an index on this table based on how we're going to query it (I anticipate `(subject, sortAt)`?).